feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package setup is scoped only to be used for scripts in path
+// feature/experimental/system/gnmi/benchmarking/ate_tests/
+// Do not use elsewhere.
+package setup
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+const (
+	// ISISInstance is ISIS instance name.
+	ISISInstance = "DEFAULT"
+	// DUTAreaAddress is DUT ISIS area address.
+	DUTAreaAddress = "49.0001"
+	// DUTSysID is DUT ISIS system ID.
+	DUTSysID = "1920.0000.2001"
+	// PeerGrpName is BGP peer group name.
+	PeerGrpName = "BGP-PEER-GROUP"
+	// DUTAs is DUT AS.
+	DUTAs = 64500
+	// ATEAs is ATE AS.
+	ATEAs = 64501
+
+	ateAs2         = 64502
+	dutStartIPAddr = "192.0.2.1"
+	ateStartIPAddr = "192.0.2.2"
+	plenIPv4       = 30
+	authPassword   = "ISISAuthPassword"
+)
+
+var (
+	// DUTIPList is DUT IP list.
+	DUTIPList = make(map[string]net.IP)
+	// ATEIPList is ATE IP list.
+	ATEIPList = make(map[string]net.IP)
+)
+
+// BuildIPList builds list of ip addresses for the ports in binding file.
+// (Both DUT and ATE ports).
+func BuildIPList(dut *ondatra.DUTDevice) {
```

Comment:
To things:
- Rather than modify global variables, it's safer to have this function return the IPLists.
- Those IP lists look to only be used by BuildBenchmarkingConfig, so I think makes sense to un-export this and have it called by BuildBenchmarkingConfig

Reply:
Changes done. Please take a look. Thanks.